training/model_management/src/run_model_preprocess.py

```diff
@@ -101,14 +105,20 @@ def _validate_pyfunc_args(pyfunc_args):
     elif mlflow_flavor == ModelFlavor.MMLAB_PYFUNC.value:
         _validate_pyfunc_args(preprocess_args)
 
-    run_preprocess(mlflow_flavor, model_path, mlflow_model_output_dir, **preprocess_args)
+    temp_output_dir = mlflow_model_output_dir / TMP_DIR
```

Comment:
Could we use a [`tempfile.TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) in the form of a context manager with the above prefix?

Reply:
updating in PR: https://github.com/Azure/azureml-assets/pull/660